packages/sysweb3-keyring/src/keyring-manager.ts

```diff
@@ -923,6 +959,86 @@ export class KeyringManager implements IKeyringManager {
     return trezorAccount;
   }
 
+  private async _createLedgerAccount(
+    coin: string,
+    slip44: string,
+    index: string,
+    label?: string
+  ) {
+    const { accounts, activeNetwork } = this.wallet;
+    let xpub;
+    try {
+      const ledgerXpub = await this.ledgerSigner.getXpub({
+        index: +index,
+        coin,
+        slip44,
+        withDecriptor: true,
+      });
+      xpub = ledgerXpub;
+    } catch (e) {
+      throw new Error(e);
+    }
+
+    const address = await this.ledgerSigner.getAddress({
+      coin,
+      index: +index,
+      slip44,
+    });
+    const accountAlreadyExists =
```

Comment:
I suggest that each part of this can be one const and `accountAlreadyExists` is the or of those consts. This way is really hard to read this variable